egs/wsj/s5/utils/lang/limit_arpa_unk_history.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+# Copyright 2018    Armin Oliya
+# Apache 2.0.
+
+# This script takes an existing ARPA lanugage model and limits the <unk> history 
```

Comment:
I think it's more standard to put this in triple-quotes as a doc-string, instead of as a comment.